sdk/avs/Azure.ResourceManager.Avs/CHANGELOG.md

```diff
@@ -1,14 +1,12 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0-beta.1 (2023-12-11)
```

Comment:
This date is the release date of .NET SDK, which should be a future date.

Reply:
updated.